src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -154,11 +177,20 @@ public static void login() {
     }
   }
 
+  private void checkForAppDefaultCredentials() {
```

Comment:
nit: this is loading the ADC into the user context, not just checking for them. Maybe something like `readAppDefaultCredentials()` or `loadAppDefaultCredentials()`?

Reply:
done